lib/src/widgets/dashboard_screen/summary.dart

```diff
@@ -18,14 +18,16 @@ class Summary extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) => Material(
-        color: Colors.white,
-        elevation: 14.0,
+        color: Colors.transparent,
+        elevation: 0.0,
         shadowColor: Color(0x802196F3),
         borderRadius: BorderRadius.circular(10.0),
         child: Column(
           children: <Widget>[
-            Container(
-              height: 300.0,
+            Padding(
+              padding: EdgeInsets.only(bottom: 30.0),
+            ),
```

Comment:
I prefer to use 8-multiple number for a size. How about `32`? And you don't need to put `.0` for this case